docs/using-in-frontend.md

```diff
@@ -0,0 +1,37 @@
+# Using nearley in browsers
+
+Use a tool like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/) to include the `nearley` NPM package in your browser code.
```

Comment:
Wait, I thought we were still in README! This is fine as-is I think. (Ignoring the fact that I personally prefer browserify :-) )